web/src/views/sleep_analysis_results/index.js

```diff
@@ -0,0 +1,154 @@
+          to which we’ve applied the fast fourier transform. We then have, for each 30 seconds epoch, the corresponding
+          amplitudes for each frequency that makes up the signal, hence the spectra. We then converted the scale to
+          logarithmic, to better see the differences in the spectrums. We then speak of signal power instead of signal
+          amplitude, because we look at the spectrums in a logarithmic scale.
+        </p>
+        <p>
+          <strong>How to read it?</strong>
+        </p>
+        <p>
+          Red therefore means that in that 30 seconds time frame, that particular frequency had a big amplitude. Green
+          means that you had that frequency with a lower amplitude. Dark blue means that you didn’t have that frequency
+          in the signal.
+        </p>
+        <p>
+          To get a better understanding at how spectrograms work, you can check out
+          <a href="https://musiclab.chromeexperiments.com/Spectrogram/" target="_blank" rel="noopener noreferrer">
+            {' '}
+            this visualization{' '}
+          </a>
+          that decomposes sound frequency from your microphone.
+        </p>
+        <SpectrogramScrollyTelling spectrograms={data.spectrograms} epochs={data.epochs} />
+        <p className="mt-5">
+          Generally, when talking about brain waves, we group certain frequencies together into bands. There are overall
+          five frequency bands, where each has a general associated behaviour, or state of mind. We will cover those
+          when looking at time frames corresponding to each sleep stage.
+        </p>
+        <p>
+          We can associate wake stages with low-amplitude activity in the 15 to 60 Hz frequency range, called the beta
+          band. By slowly falling asleep, the signal frequencies tend to decrease into the 4 to 8 Hz range, or the theta
+          band, and to have larger amplitudes. These characteristics are associated with N1. N2 stage has the same
+          characteristics, and also includes sleep spindles. They last only a few seconds and are a large oscillation in
+          the 10 to 15 hz band. Because they do not occur during all of the 30 seconds period, they cannot be seen here.
+          Stage N3, also called slow wave sleep, is characterized by slower waves between 0.5 and 4 Hz, known as the
+          delta range, with large amplitudes. REM stage has the same characteristics as Wake stage, whereas there are
+          low voltage high frequency activity.
+        </p>
+        <p>Wanna know how accurate this data is?</p>
+        <Row className="scrollytelling-container__buttons">
+          <Button className="mt-4" color="default" to="/performance" tag={Link}>
+            Check out the performances
+          </Button>
+          <Button
+            className="mt-4"
+            color="secondary"
+            href={`data:text/json;charset=utf-8,${encodeURIComponent(JSON.stringify(data.epochs))}`}
```

Comment:
C'est ici pour download le fichier

Reply:
maybe put it in a variable to self-document it?